src/nexus-prisma/NexusPrismaBuilder.ts

```diff
@@ -22,8 +17,9 @@ import {
 } from './NamingStrategies'
 import { dateTimeScalar, GQL_SCALARS_NAMES } from './scalars'
 import { getSupportedMutations, getSupportedQueries } from './supported-ops'
+import { DynamicOutputPropertyDef } from 'nexus/dist/dynamicProperty'
 
-interface NexusPrismaMethodParams {
+interface FieldPublisherConfig {
```

Comment:
I think we need a good name for what this part of the system is. I considered `Field Configurator` but then preferred `Field Publisher` because `publisher` has connotations of "exposing" something (e.g. docker port publishing) and `publish` is a much less generic/overloaded term than `config`.

`Field` is still generic here. I think we could do better. But not sure, this seems like a good step already.

Reply:
I really like `publisher`. I don't feel like `field` is being generic here, we're referring to a "GraphQL field", which AFAIK is the right terminology in their spec. Let's keep it that way!